k256/src/arithmetic/field.rs

```diff
@@ -290,14 +319,23 @@ impl ConstantTimeEq for FieldElement {
 
 impl Default for FieldElement {
     fn default() -> Self {
-        Self::zero()
+        Self::ZERO
     }
 }
 
 impl DefaultIsZeroes for FieldElement {}
 
 impl Eq for FieldElement {}
 
+impl From<u64> for FieldElement {
```

Comment:
This part I implemented a little differently in the commit above, by adding methods to specific FieldElement implementations.

Reply:
Used your implementation in [1040324](https://github.com/RustCrypto/elliptic-curves/pull/730/commits/104032401b9411086467e99e4f29fd32588fc83e)